.travis.yml

```diff
@@ -4,7 +4,7 @@ language: python
 services:
   - docker
 install:
-    - pip install pylint==1.9.2 flake8==3.5.0 requests wheel==0.30.0 -q
+    - pip install pylint==1.9.5 flake8==3.5.0 requests wheel==0.30.0 -q
```

Comment:
just quick question here, why this update was necessary?
but I am ok with that anyway, so merging